src/main/java/org/dependencytrack/event/kafka/processor/VulnerabilityScanResultProcessor.java

```diff
@@ -801,7 +801,7 @@ public interface Dao {
                 INNER JOIN
                   "PROJECT" AS "P" ON "P"."ID" = "C"."PROJECT_ID"
                 WHERE
-                  "C"."UUID" = (:uuid)::TEXT
+                  "C"."UUID" = (:uuid)::UUID
```

Comment:
Can you try if omitting the cast (`::UUID`) also works? It *should*, and if it does, I think we should omit it to make the queries less verbose.

For background, the casting was previously necessary because JDBI mapped Java `UUID`s to PostgreSQL `UUID`.